### PR TITLE
BaSP-TG-39: List Projects Page

### PR DESCRIPTION
**Pull request by FiolAxel**
- Table List
- Delete / Edit Icon
- Delete Feature
- Redirect to add form
- Warning message for delete

### CODE REVIEW

File: src/Components/Projects/index.js [hunk 1]
```diff
@@ -1,9 +1,74 @@
...
+  async function deleteProject(projectId) {
+    let confirm = window.confirm('Are you sure yo want to delete Project');
+    if (confirm) {
+      try {
+        const response = await fetch(`${process.env.REACT_APP_API_URL}/projects/${projectId}`, {
+          method: 'DELETE'
+        });
+        const data = await response.json();
+        console.log('response.json() del delete: ');
+        console.log(data);
```
Comment (CarVigil) [inline src/Components/Projects/index.js:26]: You should only remove these console.log for the response and data.

**Review by CarVigil — CHANGES_REQUESTED** [review]

**Review by vadalamati — APPROVED** [review]
Hi, Axel. Good job!!!

**Review by CarVigil — APPROVED** [review]
Very nice Axel! Everything looks fine.

**Comment by tobiaspujol** [comment]
Works fine!

**Review by laura-brussa12 — APPROVED** [review]

**Pull request merged**